srcpkgs/python3-imagesize/template

```diff
@@ -11,7 +11,11 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/shibukawa/imagesize_py"
 distfiles="${PYPI_SITE}/i/imagesize/imagesize-${version}.tar.gz"
-checksum=f3832918bc3c66617f92e35f5d70729187676313caa60c187eb0f28b8fe5e3b5
+checksum=b1f6b5a4eab1f73479a50fb79fcf729514a900c341d8503d62a62dbc4127a2b1
+
+do_check() {
+	python3 setup.py test
+}
```

Comment:
This should be done by default, and is also not the preferred approach anymore. If you drop `do_check` altogether and add `python3-pytest` to `checkdepends`, what happens?

Reply:
7 tests pass :)